guides/common/assembly_using-red-hat-insights-with-project.adoc

```diff
@@ -0,0 +1,17 @@
+include::modules/con_using-red-hat-insights-with-project.adoc[]
+
+include::modules/con_overview-of-project-integration-with-insights.adoc[leveloffset=+1]
+
+ifdef::installing-satellite-server-connected[]
+include::modules/proc_enabling-insights-on-your-project-server.adoc[leveloffset=+1]
+
+include::modules/proc_disabling-registration-to-insights.adoc[leveloffset=+1]
```

Comment:
This duplicates https://docs.redhat.com/en/documentation/red_hat_insights/1-latest/html-single/client_configuration_guide_for_red_hat_insights/index#proc-unregistering-system-insights_insights-cg-configuring-client. Rather than document how to disable the registration ourselves (and actually leaving out the details that are included in the Insights docs), I want to look into whether we could drop this section completely.